gems/aws-eventstream/lib/aws-eventstream/decoder.rb

```diff
@@ -63,26 +63,31 @@ class Decoder
       include Enumerable
 
       ONE_MEGABYTE = 1024 * 1024
+      private_constant :ONE_MEGABYTE
 
       # bytes of prelude part, including 4 bytes of
       # total message length, headers length and crc checksum of prelude
       PRELUDE_LENGTH = 12
+      private_constant :PRELUDE_LENGTH
 
       # bytes of total overhead in a message, including prelude
       # and 4 bytes total message crc checksum
-      OVERHEAD_LENGTH = 16
+      CRC32_LENGTH = 4
```

Comment:
Is this change needed?  I'd be a little worried about a user having depended on the OVERHEAD_LENGTH constant.